packages/keycodes/src/index.js

```diff
@@ -372,6 +372,18 @@ export const isKeyboardEvent = mapValues( modifiers, ( getModifiers ) => {
 			key = String.fromCharCode( event.keyCode ).toLowerCase();
 		}
 
+		// Replace some characters to match the key indicated
+		// by the shortcut on Windows.
+		if ( ! _isApple() ) {
+			if (
+				event.shiftKey &&
+				character.length === 1 &&
+				event.code === 'Comma'
+			) {
```

Comment:
I have changed the process to replace only certain keys so that it does not affect the keyboard layout.
I have nested the if statements in anticipation of replacing other keys in the future.
Additionally, as mentioned in [this comment](https://github.com/WordPress/gutenberg/pull/43428#pullrequestreview-1109036033), I excluded AppleOS.